lib/src/base-styles/_nx-lists.scss

```diff
@@ -171,59 +167,84 @@
 /*
  * Action or Clickable Lists
  */
-.nx-list--clickable {
-  .nx-list__item {
 
-    &, > .nx-list__link {
-      &.selected {
-        &, &:hover {
-          background-color: var(--nx-color-interactive-background-selected);
-        }
-      }
+.nx-list__item--link, .nx-list__item--clickable {
+  display: contents;
+}
 
-      &:hover {
-        background-color: var(--nx-color-interactive-background-hover);
-        cursor: pointer;
+.nx-list__item--clickable, // preferred
+.nx-list__item--link,
+.nx-list--clickable .nx-list__item // deprecated
+{
+  &, > .nx-list__link, > .nx-list__btn {
+    &.selected {
+      &, &:hover {
+        background-color: var(--nx-color-interactive-background-selected);
       }
+    }
 
-      &:active {
-        background-color: var(--nx-color-interactive-background-active);
-      }
+    &:hover {
+      background-color: var(--nx-color-interactive-background-hover);
+      cursor: pointer;
+    }
 
-      &:focus {
-        box-shadow: var(--nx-box-shadow-focus);
-        border-color: var(--nx-color-interactive-border-focus);
-        outline: none;
-      }
+    &:active {
+      background-color: var(--nx-color-interactive-background-active);
+      color: inherit;
```

Comment:
I think we need this all the time not just on `:active`.  It seems that inactive ones get pure black as their text color (user-agent default button style)

Reply:
> I think we need this all the time not just on `:active`. It seems that inactive ones get pure black as their text color (user-agent default button style)

Moving `color: inherit;` to the top of these rules so that it covers links and buttons fixed it as well as for all interactive states. https://github.com/sonatype/sonatype-react-shared-components/pull/226/commits/f7894c0f9939a5543d848d6e4f8abb25b89b030b